spk/tt-rss/src/dsm-control.sh

```diff
@@ -10,71 +10,74 @@ WEB_DIR="/var/services/web"
 PATH="${INSTALL_DIR}/bin:${PATH}"
 BUILDNUMBER="$(/bin/get_key_value /etc.defaults/VERSION buildnumber)"
 USER="$([ "${BUILDNUMBER}" -ge "4418" ] && echo -n http || echo -n nobody)"
-PHP="$([ "${BUILDNUMBER}" -ge "7321" ] && echo -n /usr/local/bin/php56 || echo -n /usr/bin/php)"
+
 TTRSS="${WEB_DIR}/${PACKAGE}/update.php"
 PID_FILE="${INSTALL_DIR}/var/tt-rss.pid"
 
-
 start_daemon ()
 {
-    start-stop-daemon -S -q -m -b -N 10 -x ${PHP} ${TTRSS} -c ${USER} -u ${USER} -p ${PID_FILE} \
-      -- --daemon 2> /dev/null
+	source "${INSTALL_DIR}/bin/php-settings.sh"
+	
+	start-stop-daemon -S -q -m -b -N 10 -x "${php}" \
+		-c ${USER} -u ${USER} -p ${PID_FILE} --  \
+		-c "${php_configuration_file}" $(compute_php_options) \
+		${TTRSS} --daemon 2>/dev/null
 }
 
 stop_daemon ()
 {
-    start-stop-daemon -K -q -u ${USER} -p ${PID_FILE}
-    wait_for_status 1 20 || start-stop-daemon -K -s 9 -q -p ${PID_FILE}
+	start-stop-daemon -K -q -u ${USER} -p ${PID_FILE}
+	wait_for_status 1 20 || start-stop-daemon -K -s 9 -q -p ${PID_FILE}
```

Comment:
May you please revert spaces-to-tabs replace on the whole file. Thanks in advance

Reply:
Done.